api/views.py

```diff
@@ -40,6 +40,12 @@ def bad_http_request(header, message):
     return HttpResponse('<h2>%s</h2><p>%s</p>' % (header, message), status=400)
 
 
+def bad_form_request(field_name, message, statusCode=400):
+    return JsonResponse({'errors': [{
+        'field_name': field_name,
+        'error_message': message}]}, status=statusCode)
+
```

Comment:
@frozenhelium Is this format okay for the client-side?

Reply:
I don't think this is the exact format we want. Let's hold on merging until we can discuss this with @frozenhelium and @tnagorra - but, yes, hopefully that's just a matter of changing the JSON structure in this one function - otherwise this looks good!

Reply:
This looks good for now.